src/modules/roc_audio/mixer.h

```diff
@@ -67,6 +74,7 @@ class Mixer : public IReader, public core::NonCopyable<> {
     core::Slice<sample_t> temp_buf_;
 
     bool valid_;
+    const size_t frame_size_;
```

Comment:
It seems that we need frame_size_ only in ctor. Thus we can make it a local variable in ctor and don't add a new field. Probably the same is true for some other classes too.

Reply:
done.